src/kepler_model/estimate/model_server_connector.py

```diff
@@ -54,10 +48,10 @@ def unpack(energy_source, output_type, response, replace=True):
     return output_path
 
 
-def make_request(power_request):
+def make_request(power_request, machine_spec):
     if not is_model_server_enabled():
         return None
-    model_request = make_model_request(power_request)
+    model_request = make_model_request(power_request, machine_spec)
     output_type = ModelOutputType[power_request.output_type]
     try:
         response = requests.post(get_model_server_req_endpoint(), json=model_request)
```

Comment:
Unrelated but .. FYI since I see that we are using POST here. POSTs are meant to make some changes to the server. 
https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/POST these are unsafe methods and thus won't be  cached by any intermediaries.

Reply:
Oh I see. Yes, agree that we should change to GET. But this needs to change in both estimator and kepler side. Let's do that later in a graceful way.